lib/cache_helper.rb

```diff
@@ -0,0 +1,10 @@
+# note, if you modify this file you have to restart the server or console
+module CacheHelper
+  def skin_cache_key(skin)
+    'skins_generation/' + (skin.type.nil? ? ("site_skin") : skin.id.to_s)
+  end
+
+  def skin_cache(skin)
+    Rails.cache.increment(skin_cache_key(skin))
+  end
+end
```

Comment:
The idea was that we could put other cache helpers in there so the number of files did not escalate.